projects/addhub.md

```diff
@@ -0,0 +1,49 @@
+Addhub
+--------------------
+
+### Background:
+We are going to set a user-defined advertisement push/pull platform.
+
+The basic function of the platform:
+user log into the platform, post his/her advertisement with keywords, such as:
+"I would like to sell a sofa.I am Brooklyn. The sofa price is $50."
+"keywords: sofa, brooklyn, $50".
+Then he could choose which public platforms he'd like to push, such as craiglist and facebook second forum and local buzzsell forum.
+The posted advertisement will be pushed to multiple web platforms.
+
+He can also choose how frequently he'd like to refresh the post, so as to keep the post information at the top of discussion board.
+
+An advanced feature is that above is a feature of seller. While for buyer, he can log in the platform, apply key words he concerns, then our platform will fetch related information from public platforms. For exampe:
+if a user like to buy a book, named 'Othello', with price $10. He can apply keywords as : 'book' 'Othello' '$(3~10)' 'Brooklyn' 
+Then our platform will fetch all posted information maximum matched the keywords and push back to user regurlarly. He would not need to visit craiglist a lot, wasting time and effort, and won't miss related post.
+
+https://github.com/addhub 
+
+### Challenge:
+a. KPI validity. Some website restrict on using API to push/pull information.
+b. An attractive application design is still pending for second phase development.
+c. Naturual language processing technogy -  If the public platformd don't provide keywords functions, then fetching related post information back to users will depends on how NLP process forum contents with proper keywords that the users desire to search for.
+
+
+### Team
+* Jacqueline Pang https://github.com/Cornell-CS5356-Fall2015/cs5356/blob/master/people/jacqueline-pang.md
+* Sasinda Premarathna https://github.com/Cornell-CS5356-Fall2015/cs5356/blob/master/people/sasinda-rukshan.md
+* Yilin Xu https://github.com/Cornell-CS5356-Fall2015/cs5356/blob/master/people/yilin-xu.md
+
+### First Story
+* https://github.com/addhub/prototype/issues/1
+- As a person I want to go to addhub site and add an advertisement to sell something I want to sell. 
```

Comment:
You don't need to repeat a story here. However the story is more like the "whole thing". A story should be very very basic and something you can implement in a week.